test/framework/cluster_proxy.go

```diff
@@ -456,3 +456,13 @@ func (p *clusterProxy) Dispose(ctx context.Context) {
 		}
 	}
 }
+
+// IsEnvWSL checks if the current environment is Windows Subsystem for Linux (WSL).
+// If so, with the Docker provider, we could not access the cluster with docker network.
+func IsEnvWSL() bool {
```

Comment:
```suggestion
func isEnvWSL() bool {
```

Let's not export this.

Reply:
I'm using this func in pkg `kubetest`, maybe we could just remove the function and use `os.Getenv("WSL_DISTRO_NAME") != "" ` in pkg `kubetest`??

Reply:
+1. Sometimes copying is better than a dependency (+ exposing a public func)